File: admin/src/main/java/top/zzf4/blog/entity/dto/ArticleDetailDTO.java
package top.zzf4.blog.entity.dto;

import lombok.Data;
import top.zzf4.blog.entity.model.Category;

import java.time.LocalDateTime;

@Data
public class ArticleDetailDTO {
    private Long id;

    private String title;

    private String content;

    private Category category;

    private LocalDateTime publishDate;

    private LocalDateTime updateTimee;

    private int views;

    private byte wordNum; // TINYINT映射为byte
}

File: blog-backend/internal/dto/article_detail_dto.go
package dto

import (
	"time"
)

type ArticleDetailDTO struct {
	ID          int64     `json:"id"`
	Title       string    `json:"title"`
	Content     string    `json:"content"`
	Category    Category  `json:"category"`
	PublishDate time.Time `json:"publish_date"`
	UpdateTime  time.Time `json:"update_time"`
	Views       int       `json:"views"`
	WordNum     byte      `json:"word_num"`
}
